R/template.R

```diff
@@ -1,110 +1,184 @@
 #'
 #' @details
 #'
-#' `metric_summarizer()` is generally called from the data frame version
+#' `numeric_metric_summarizer()`, `class_metric_summarizer()`, and
+#' `prob_metric_summarizer()` are generally called from the data frame version
 #' of your metric function. It knows how to call your metric over grouped data
 #' frames and returns a `tibble` consistent with other metrics.
 #'
 #'
-#' @param metric_nm A single character representing the name of the metric to
+#' @param name A single character representing the name of the metric to
 #' use in the `tibble` output. This will be modified to include the type
 #' of averaging if appropriate.
 #'
-#' @param metric_fn The vector version of your custom metric function. It
+#' @param fn The vector version of your custom metric function. It
 #' generally takes `truth`, `estimate`, `na_rm`, and any other extra arguments
 #' needed to calculate the metric.
 #'
 #' @param data The data frame with `truth` and `estimate` columns passed
 #' in from the data frame version of your metric function that called
-#' `metric_summarizer()`.
+#' `numeric_metric_summarizer()`, `class_metric_summarizer()`, or
+#' `prob_metric_summarizer()`
 #'
 #' @param truth The unquoted column name corresponding to the `truth` column.
 #'
 #' @param estimate Generally, the unquoted column name corresponding to
 #' the `estimate` column. For metrics that take multiple columns through `...`
 #' like class probability metrics, this is a result of [dots_to_estimate()].
 #'
-#' @param estimator For numeric metrics, this is left as `NULL` so averaging
-#' is not passed on to the metric function implementation. For classification
-#' metrics, this can either be `NULL` for the default auto-selection of
-#' averaging (`"binary"` or `"macro"`), or a single character to pass along
-#' to the metric implementation describing the kind of averaging to use.
+#' @inheritParams rlang::args_dots_empty
+#'
+#' @param estimator This can either be `NULL` for the default auto-selection of
+#' averaging (`"binary"` or `"macro"`), or a single character to pass along to
+#' the metric implementation describing the kind of averaging to use.
 #'
 #' @param na_rm A `logical` value indicating whether `NA` values should be
 #' stripped before the computation proceeds. The removal is executed in
 #' `metric_vec_template()`.
 #'
-#' @param event_level For numeric metrics, this is left as `NULL` to prevent
-#' it from being passed on to the metric function implementation. For
-#' classification metrics, this can either be `NULL` to use the default
-#' `event_level` value of the `metric_fn` or a single string of either
-#' `"first"` or `"second"` to pass along describing which level should be
-#' considered the "event".
+#' @param event_level This can either be `NULL` to use the default `event_level`
+#' value of the `fn` or a single string of either `"first"` or `"second"`
+#' to pass along describing which level should be considered the `"event"`.
```

Comment:
```suggestion
#' to pass along describing which level should be considered the "event".
```

Purposeful to not use backticks here